content/ru/docs/concepts/workloads/_index.md

```diff
@@ -0,0 +1,101 @@
+Независимо от того, представляет ли ваша рабочая нагрузка один компонент
+или несколько, которые работают вместе, в Kubernetes вы 
+запускаете ее внутри набора [_подов_](/docs/concepts/workloads/pods).
+В Kubernetes под представляет собой набор работающих
+{{< glossary_tooltip text="контейнеров" term_id="container" >}} в вашем кластере.
+
+Поды Kubernetes имеют определенный [жизненный цикл](/docs/concepts/workloads/pods/pod-lifecycle/).
+Например, если у вас запущен под в кластере, то критическая ошибка на 
+{{< glossary_tooltip text="узле" term_id="node" >}}, где этот модуль работает,
+означает, что все поды на этом узле выходят из строя. Kubernetes считает
+этот уровень сбоя безвозвратным: вам нужно будет создать новый под для восстановления,
+даже если узел позже станет работоспособным.
+
+Однако, чтобы значительно облегчить жизнь, вам не нужно 
+непосредственно контролировать каждый под. Вместо этого вы можете использовать
+_ресурсы рабочей нагрузки_ которые управляют набором подов от вашего имени.
+Эти ресурсы настраивают {{< glossary_tooltip term_id="controller" text="контроллеры" >}},
+которые обеспечивают запуск нужного количества модулей
+нужного типа в соответствии с указанным вами состоянием.
+
+Kubernetes предоставляет несколько встроенных ресурсов для рабочих нагрузок:
+
+* [Deployment](/docs/concepts/workloads/controllers/deployment/) и [ReplicaSet](/docs/concepts/workloads/controllers/replicaset/)
+  (замена устаревшего ресурса
+  {{< glossary_tooltip text="ReplicationController" term_id="replication-controller" >}}).
+  Deployment хорошо подходит для управления неизменными приложениями
+  в вашем кластере, где любой под в deployment'е взаимозаменяем
+  и может быть заменен при необходимости.
+* [StatefulSet](/docs/concepts/workloads/controllers/statefulset/) позволяет запускать один или несколько
+  связанных подов, которые как-то отслеживают состояние. Например, если ваше
+  приложение записывает постоянные данные, вы можете использовать StatefulSet,
+  который сопоставляет каждый под с [PersistentVolume](/docs/concepts/storage/persistent-volumes/).
+  Ваш код, работающий в подах этого StatefulSet, может 
+  копировать данные в другие поды в том же StatefulSet,
+  чтобы повысить общую отказоустойчивость.
+* [DaemonSet](/docs/concepts/workloads/controllers/daemonset/) создает поды, которые
+  предоставляют инструменты, локальные для узлов.
+  Каждый раз, когда вы добавляете в кластер узел,
+  соответствующий спецификации DaemonSet,
+  плоскость управления (control plane) планирует под для этого DaemonSet 
+  на новом узле. Каждый под в DaemonSet выполняет работу,
+  аналогичную работе системного демона на классическом сервере Unix/POSIX.
+  DaemonSet может иметь основополагающее значение для работы вашего кластера,
+  например, плагин для запуска сети кластера ([cluster networking](/docs/concepts/cluster-administration/networking/#how-to-implement-the-kubernetes-network-model)),
```

Comment:
Whenever we have a localised page, we need to use a corresponding link:
```suggestion
  например, плагин для запуска сети кластера ([cluster networking](/ru/docs/concepts/cluster-administration/networking/#реализация-сетевой-модели-kubernetes)),
```

Reply:
Ah good catch, I missed this one. Thank you!